backend/inspirehep/records/tasks.py

```diff
@@ -43,3 +54,66 @@ def update_records_relations(uuids):
 
     db.session.commit()
     return uuids
+
+
+@shared_task(
+    bind=True,
+    queue="indexer_task",
+    acks_late=True,
+    retry_backoff=True,
+    retry_kwargs={"max_retries": 3},
+    autoretry_for=(OperationalError, TransportError),
+)
+def redirect_references_to_merged_record(self, uuid):
+    record = InspireRecord.get_record(uuid)
+    deleted_record_refs = record.get_new_deleted_records()
+    if deleted_record_refs:
```

Comment:
better avoid unneeded indentation
```suggestion
    if not deleted_record_refs:
        return
    # the rest
```